frontend/src/components/GraphContainer.jsx

```diff
@@ -0,0 +1,27 @@
+import Graph from './Graph.jsx';
+import FutureDocument from './FutureDocument.jsx';
+import Col from 'react-bootstrap/Col';
+import Row from 'react-bootstrap/Row';
+
+function GraphContainer({data, createOn, setLastUpdate, backend, user}) {
+  const docs = data
+    .map((key) => [key._id, key.dc_title, key.links]);
+  const displayedDocs = docs.flatMap(d => d[0]);
+
+  return (
+    <>

```

Comment:
An empty elements is necessary in JSX only when the HTML fragment contains more than one root element.